src/types/profile.ts

```diff
@@ -28,7 +28,7 @@ export const toInitializedProfile = (
   blocked_inbox_or_channels: profile.blocked_inbox_or_channels,
   date_of_birth:
     user.date_of_birth !== undefined
-      ? formatDate(user.date_of_birth)
+      ? new Date(formatDate(user.date_of_birth))
```

Comment:
@gunzip please double-check this change as we need to be sure that the type-change is invariant for the API behaviour.

Reply:
can you try to test this locally using io-mock?

Reply:
basically the output format now changes from `1991-12-12` to `1991-12-12T00:00:00.000Z`. The client app ignores this value anyway.